src/utilities/include/utils.h

```diff
@@ -18,6 +18,11 @@
 #include <mutex>
 #include <unordered_map>
 
+#if defined(_MSC_VER)
+#include <BaseTsd.h>
+typedef SSIZE_T ssize_t;
```

Comment:
This makes me slightly worried about `ssize_t`...

Reply:
yeah, thinking about it `ssize_t` is not really needed.